api/s2n.h

```diff
@@ -249,13 +249,11 @@ typedef enum {
 /**
  * Determines whether s2n-tls is operating in FIPS mode.
  *
- * s2n-tls enters FIPS mode on initialization when the linked libcrypto has FIPS mode enabled. Some
- * libcryptos, such as AWS-LC-FIPS, have FIPS mode enabled by default. With other libcryptos, such
- * as OpenSSL, FIPS mode must be enabled before initialization by calling `FIPS_mode_set()`.
+ * s2n-tls enters FIPS mode on initialization when linked with a FIPS validated version of AWS-LC.
  *
- * s2n-tls MUST be linked to a FIPS libcrypto and MUST be in FIPS mode in order to comply with FIPS
- * requirements. Applications desiring FIPS compliance should use this API to ensure that s2n-tls
- * has been properly linked with a FIPS libcrypto and has successfully entered FIPS mode.
+ * s2n-tls MUST be linked to a FIPS validated libcrypto and MUST be in FIPS mode in order to comply
+ * with FIPS requirements. Applications desiring FIPS compliance should use this API to ensure that
+ * s2n-tls has been properly linked with a FIPS libcrypto and has successfully entered FIPS mode.
  *
```

Comment:
Oh wait I worry that this makes it sound like those are the only two requirements. Should we also add that the application needs to use a FIPS security policy? Or maybe add a little warning that even if s2n-tls is in fips mode, it can use non-fips algorithms unless properly configured with a security policy that only offers fips algorithms?

Reply:
updated in https://github.com/aws/s2n-tls/pull/4857/commits/0a9f60c588b500771ab83b7763d8343031bc4b5a